packages/twitch-eventsub/src/Events/EventSubChannelRewardAddEvent.ts

```diff
@@ -17,16 +18,16 @@ export interface EventSubChannelRewardAddEventData {
 	redemptions_redeemed_current_stream: number | null;
 	max_per_stream: EventSubMaxStreamData;
 	max_per_user_per_stream: EventSubMaxStreamData;
-	global_cooldown: EventSubCooldownData; // eslint-disable-line @typescript-eslint/naming-convention
+	global_cooldown: EventSubCoolDownData;
 	background_color: string;
 	image: EventSubRewardImageData;
-	default_image: EventSubRewardImageData;
+	default_image: Required<EventSubRewardImageData>;
```

Comment:
I actually read the spec wrong here, the whole `image` property is actually nullable in case of no images uploaded. The `default_image` property is the same data type, but not nullable. This structure also matches the Helix `getCustomRewards` endpoint. I'll add this change to the PR.